tests/api/auth/signup.test.js

```diff
@@ -56,17 +52,42 @@ describe('auth:signup', () => {
 })
 
 describe('auth:username-availability', () => {
-  it('should reject an account with already created username', async () => {
+  it('should reject an account with an already used username', async () => {
     const username = createUsername()
     await createUser({ username })
     await wait(10)
-    await publicReq('post', endpoint, {
-      username,
-      email: `bla${username}@foo.bar`,
-      password: randomString(8)
+    await signup({ username })
+    .then(shouldNotBeCalled)
+    .catch(err => {
+      err.body.status_verbose.should.equal('an account is already in the process of being created with this username')
     })
+  })
+
+  it('should normalize unicode letters', async () => {
```

Comment:
as every string is formatted/normalized in the params sanitize, i think this test can be removed (which, i bet it was useful for development). 
Otherwise, every api tests involving a string parameter should make sure unicode is also normalized

Reply:
tests for other endpoints could be added, but it is much more critical for signup, due to username uniqueness constrain; I would thus not mind having only tests for endpoints where normalization is critical

Reply:
but this specific string is also normalized in the user model, so we could afford to not have one more api test if an integration test can do the job.
my point is that this test does not improve the test coverage, but well i guess this can be non blocking..